application/config.json.template

```diff
@@ -44,5 +44,45 @@
            "Other"
        ],
        "categoryRoleSuffix": " - Helper"
-   }
+   },
+    "blacklistedFileExtension": [
+        "exe",
+        "pif",
+        "application",
+        "gadget",
+        "msi",
+        "msp",
+        "com",
+        "scr",
+        "hta",
+        "cpl",
+        "msc",
+        "jar",
+        "bat",
+        "cmd",
+        "vb",
+        "vbs",
+        "vbe",
+        "js",
+        "jse",
+        "ws",
+        "wsf",
+        "wsh",
+        "wsc",
+        "ps1",
+        "ps1xml",
+        "ps2",
+        "ps2xml",
+        "psc1",
+        "psc2",
+        "msh",
+        "msh1",
+        "msh2",
+        "mshxml",
+        "msh1xml",
+        "msh2xml",
+        "scf",
+        "lnk",
+        "inf"
```

Comment:
Some of these shouldnt be on that list, as they are commonly shared in servers like ours and would lead to too many false positives:
* jar
* js

Also, could u sort the list alpahbetically? Easier to work with then, thanks.

Reply:
If the idea was to make the server safer, I think all executables should go. Someone posting an executable is a big red flag for me, and wouldn't be comfortable letting people test it out on their machines. Especially since we can't (or won't) verify that it's actually safe for our members to run. It could be a keylogger, and we would never know. And we kinda implicitly stand behind it if we allow it, basically hosting a virus.

Also, I don't see a situation where posting jars would be needed, or even wanted. It's a minor inconvenience for the user posting the jar, and that's all, doubt anyone would want to run it anyway. So it's a high risk, low benefit kind of thing from my perspective.

Reply:
I wouldn't mind letting trusted users post jars tho. Or if we had a process where people first run the jar with us, to make sure it's safe, and then they can post with the permission.

Or anything similar really, my argument is that it shouldn't be allowed by default, since it's dangerous and abuse potential is high. And it's not like something our members want/need to do, we had maybe few jars in the past year.

Reply:
well i would be more on the defensive side, just dont allow any executable. If there is rly so much need to post .jars then maybe we can add "trusted user" which can post .jars and so on.

Reply:
There is a need of posting jars. it happens about twice per month. For example users have trouble with their jar, it just doesnt start or similar. Then we ask them to just upload the jar and we can check it.

As a minor side note, Discord already has a built in "potentially malicious, do you really want to download?!" warning on all executable files - so we are building a second layer of defense here, not the first.

Since we are helping with jar development on our server, we should also allow people to throw jar around. Otherwise they just escape the help-conversation into DM and then we cant protect them at all anymore.

Reply:
Ok then the config dont need an update